channels/iq.m3u

```diff
@@ -89,14 +89,10 @@ http://31.14.40.139:1935/liveTrans/ngrp:myStream2_all/playlist.m3u8
 http://163.172.39.215:25461/live/line/C4@!a3a1@!w72A/30.m3u8
 #EXTINF:-1 tvg-id="KurdMax.iq" tvg-name="KurdMax" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="https://kurdmaxcms.com/asset/image/footer_logo/%D9%A1_1.png" group-title="",KurdMax (1080p) [Not 24/7]
 http://163.172.39.215:25461/live/line/C4@!a3a1@!w72A/31.m3u8
-#EXTINF:-1 tvg-id="KurdMax.iq" tvg-name="KurdMax" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="https://kurdmaxcms.com/asset/image/footer_logo/%D9%A1_1.png" group-title="",KurdMax (1080p) [Offline]
-https://5aa8c7549bfa9.streamlock.net/liveTrans/ngrp:myStream_all/playlist.m3u8
```

Comment:
Works for me

![image](https://user-images.githubusercontent.com/30985701/131243515-90f5d9a1-6e58-4e37-987a-032ccf5ffad2.png)